packages/f-content-cards/src/components/cardTemplates/VoucherCard.vue

```diff
@@ -97,4 +169,29 @@ export default {
         color: $color-link-default;
         text-align: right;
     }
+
+    .c-contentCard-voucher-copy-copied {
+        color: $orange--aa;
+    }
+
+    .c-contentCard-voucher-copy-transition-out {
+        transition: color 1s;
+    }
+
+    .c-contentCard-voucher-copy-cooldownTick-leave-to {
+        opacity: 0;
+    }
+
+    .c-contentCard-voucher-copy-cooldownTick-leave-active {
+        transition: opacity 1s;
+    }
+
+    .c-contentCard-voucher-code-cooldown-tick {
+        // background-color: $color-text--success;
```

Comment:
Should this be removed?

Reply:
yes thanks for the spot, also sorted out the rebase mess